File: src/ChickenEgg.cpp
#include "ChickenEgg.hpp"

ChickenEgg::ChickenEgg(){
    harga = 5000;
}

ChickenEgg::ChickenEgg(int h) {
    harga = h;
}

string ChickenEgg::toString(){
    return "Chicken Egg";
}

File: src Java/ChickenEgg.java
public class ChickenEgg extends FarmProduct {
    /**
     * Default constructor
     */
    public ChickenEgg(){
        setHarga(5000);
    }

    /**
     * User-defined constructor
     * @param h nilai harga yang diinginkan
     */
    public ChickenEgg(int h){
        setHarga(h);
    }

    /**
     * Nama objek
     * @return "Chicken Egg"
     */
    public String toString(){
        return "Chicken Egg";
    }
}
